### PR TITLE
Upload page - Ethereum for .NET Developers

### PR DESCRIPTION
**Pull request by Enigmatic331**
Materials for application developers to quickly start developing on Ethereum using the .NET platform. Also touches on Nethermind, an Ethereum client built with .NET Core.

## Description
Initial upload of the .NET specific page for ethereum.org.

## Related Issue
https://github.com/ethereum/ethereum-org-website/issues/267

## Checklist:

- [x] I have read the contributing guidelines in the project README.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/dotnet/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+- [Understanding Smart Contracts](https://kauri.io/article/e4f66c6079e74a4a9b532148d3158188/ethereum-101-part-5-the-smart-contract)
+- [Write your First Smart Contract](https://kauri.io/article/124b7db1d0cf4f47b414f8b13c9d66e2/remix-ide-your-first-smart-contract)
+- [Learn How to Compile and Deploy Solidity](https://kauri.io/article/973c5f54c4434bb1b0160cff8c695369/understanding-smart-contract-compilation-and-deployment)
+
+## Beginner References and Links
+
+**Introducing the Nethereum library and VS Code Solidity**
+
+- [Nethereum, Getting Started](https://docs.nethereum.com/en/latest/getting-started/)
+- [Installing VS Code Solidity](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity/)
```
[the remark this answers is not in the document]
Comment (Enigmatic331) [reply docs/dotnet/index.md]: Done and done - @samajammin, thanks so much for taking the time to review and catching this!

**Comment by Enigmatic331** [comment]
> 
> 
> This is great @Enigmatic331 - thanks for the changes!
> 
> I've merged #360, which was the first PR for this issue. Since your PR has valuable additions on top of #360, we'd still like to incorporate your contributions & reward you for the bounty. Please rebased off the current `dev` branch & incorporate your content into the existing file (`docs/dot-net/index.md`). Please be sure to delete any duplicate references.
> 
> Thanks again!

Gosh - I wasn't expecting this, thanks a ton for the bounty fee! Am humbled.

I will work this out over the next couple of hours. Also will be periodically checking back (as promised: https://twitter.com/EnigmaticUnreal/status/1177612971372994561) to make sure contents are up to date, new contents get posted, etc.

**Comment by samajammin** [comment]
Ah, yes! I remember that tweet - that was the original impetus for creating this issue. Glad you came to solve it 😄 

We do hope you continue to contribute & keep this page up to date! We're always on the lookout for potential maintainers for various parts of the website & we're in need of .NET gurus.

**Comment by samajammin** [comment]
@Enigmatic331 - I'm closing this out in favor of your #531 PR. Please focus on the changes requested there.

**Pull request closed without merging**